From 44d0703b61156a19c6cbfb5cfe3f5d58e328daae Mon Sep 17 00:00:00 2001
From: Leah Rowe <info@minifree.org>
Date: Mon, 2 Dec 2024 11:16:43 +0000
Subject: [PATCH 1/1] Disable m2 hotplug in lenovo t480s devicetree

This fixes a bug where nvme disappears and gets renamed on
s3 resume.

Thanks go to Mate Kukri who suggested this.

Signed-off-by: Leah Rowe <info@minifree.org>
---
 .../lenovo/sklkbl_thinkpad/variants/t480s/overridetree.cb        | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/mainboard/lenovo/sklkbl_thinkpad/variants/t480s/overridetree.cb b/src/mainboard/lenovo/sklkbl_thinkpad/variants/t480s/overridetree.cb
index 2cac8c4a75..9129d3a1b8 100644
--- a/src/mainboard/lenovo/sklkbl_thinkpad/variants/t480s/overridetree.cb
+++ b/src/mainboard/lenovo/sklkbl_thinkpad/variants/t480s/overridetree.cb
@@ -115,7 +115,6 @@ chip soc/intel/skylake
 			register "PcieRpClkSrcNumber[8]"		= "5"
 			register "PcieRpAdvancedErrorReporting[8]"	= "1"
 			register "PcieRpLtrEnable[8]"			= "1"
-			register "PcieRpHotPlug[8]"			= "1"
 		end
 	end
 end
-- 
2.39.5

